src/dbnode/persist/fs/files_test.go

```diff
@@ -34,9 +34,9 @@ import (
 	"time"
 
 	"github.com/m3db/m3/src/dbnode/digest"
+	"github.com/m3db/m3/src/dbnode/namespace"
```

Comment:
Did you have a benchmark for the new parsing function? Would be cool to have that in source control if you still have it.

Reply:
I think it was very simple like this with a bunch of different inputs:
```go
func BenchmarkFilenameParsing(b *testing.B) {
	for n := 0; n < b.N; n++ {
		ts, i, err := TimeAndVolumeIndexFromFileSetFilename("foo-1-0-data.db")
		_, _, _ = ts, i, err
	}
}
```
Not too valuable to include probably? Just wanted to verify no allocs and that it was significantly faster than the previous implementation.